### PR TITLE
Advanced Mimery tweak

### PR DESCRIPTION
**Pull request by TrustyGun**
:cl:
tweak: Traitor Mimes with the finger guns spell now fire 3 bullets at a time, as opposed to just 1.
/:cl:

Finger Guns beforehand, with it's long charge time, did not make up for it with one measly detective bullet. With this change, Mimes get more bang for their buck by shooting 3 bullets at a time.

### CODE REVIEW

File: code/modules/uplink/uplink_item.dm [hunk 1]
```diff
@@ -1222,7 +1222,6 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	desc = "A tattered old jumpsuit that will provide absolutely no benefit to you. It fills the wearer with a strange compulsion to blurt out 'glorf'."
 	item = /obj/item/clothing/under/color/grey/glorf
 	cost = 20
-	surplus = 0
```
Comment (TrustyGun) [inline code/modules/uplink/uplink_item.dm]: surplus items are already at _surplus = 0_, so this line was useless.

**Comment by kevinz000** [comment]
no-inhand stun bullets seem a bit overpowered. why not make them higher-burst, lower cooldown, but with low non-stun damage so it's actually a gun that isn't just used to stun people?

**Comment by TrustyGun** [comment]
i dont see how this is overpowered. this is still weaker than every other gun in the game due to the cooldown. for example, the detective can shoot more than 10 bullets way quicker than 30 seconds

**Comment by CPTANT** [comment]
Well there is that part where this is totally undetectable.

**Comment by CPTANT** [comment]
And you also get 3x1 walls, which are actually pretty OP.

**Comment by TrustyGun** [comment]
Well I still feel that Mime Guns need a little push so they are worth it for the price

File: code/modules/uplink/uplink_item.dm (hunk 1 above)
Comment (coiax) [inline code/modules/uplink/uplink_item.dm]: except we might want to make some job specific items a low level surplus, and a mimery kit is always useless for non-mimes. So keep it.

Comment (TrustyGun) [inline code/modules/uplink/uplink_item.dm]: this isnt the mime kit, its the glorf suit

Comment (coiax) [inline code/modules/uplink/uplink_item.dm]: Glorf suit is a useless 20tc item that should always be barred from surplus crate as well

Comment (TrustyGun) [inline code/modules/uplink/uplink_item.dm]: I added it back to the glorf suit, and added it to the mime kit now.

**Comment by KorPhaeron** [comment]
if it ends up stupid we can just nerf it

**Pull request merged**